packages/next-drupal/src/client.ts

```diff
@@ -1359,14 +1359,23 @@ export class DrupalClient {
 
     const basic = Buffer.from(`${clientId}:${clientSecret}`).toString("base64")
 
-    const response = await fetch(url.toString(), {
+    const options = {
```

Comment:
For consistency, this is called `init` in the `this.fetch` because options refers to something else.

Reply:
This is updated!